### ISSUE TITLE
A11y_code.visualstudio.com_Docs Node.JS/Javascript_Node.js Debugging_ScreenReader: Screen reader is not announcing status message displayed after submitting feedback.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#2814]
“[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details**:
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 4.1.3 (WCAG 4.1.3)

**Repro Steps**:
Navigate to the URL: [https://code.visualstudio.com/docs/nodejs/nodejs-debugging](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)
follow below steps:
1.Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "Docs" link.
3.From left navigation, select "Node.JS/Javascript".
4.Navigate to "Node.js Debugging" link.
5.Start the screen reader.
6.Navigate to feedback section and click on Submit feedback button.

**Actual**:
When we  hit on "Submit" or "Skip this" button, there is message "Thanks You" appears on screen and disappear after few seconds. That status message is not announced by screen reader.
  
![image](https://user-images.githubusercontent.com/52035395/60156078-4d44e880-9809-11e9-8cad-03c2a3f5dbf9.png)

This issue is observed with Edge+Narrator and Chrome+NVDA

**Expected**:
Screen reader should announce status message after submitting or skipping feedback.

**Recommendations**:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact**:
Screen reader user will not understand that there is any message displayed on screen.

**MAS Reference**:
MAS 4.1.3 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=f1ecca2d-d172-4d08-9d40-6fd75b5d2505](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=f1ecca2d-d172-4d08-9d40-6fd75b5d2505)

**Attachment for Reference**:
[Code.visualstudio.com_Node.js Debugging_Status Message.zip](https://github.com/microsoft/vscode-docs/files/3329629/Code.visualstudio.com_Node.js.Debugging_Status.Message.zip)

**Comment by RohitJadhav-TCS** [comment]
Similar issue is observed in through out application where the Feedback section is present.

**Comment by gregvanl** [comment]
Closing for now since the WCAG 2.1 standard is not currently required for Grade C.

**Comment by SwatiTiwari-TCS** [comment]
"The issue is still reproducing can you please open the issue".

**Comment by gregvanl** [comment]
@SwatiTiwari-TCS A recent change keeps the Thank you message on the screen. Is that not sufficient?
This change was done for. https://github.com/microsoft/vscode-docs/issues/3586.

**Comment by mstechie** [comment]
GitHubTags:#A11yMAS;
